Engine/source/gui/controls/guiTextCtrl.cpp

```diff
@@ -188,7 +188,7 @@ void GuiTextCtrl::setText(const char *txt)
    if( !mProfile )
       return;
    
-   if (txt)
+   if (txt && txt != mText)
```

Comment:
Right, I thought of that afterwards then thought 'nah that's crazy, who'd ever do that!' Maybe add a comment for future generations?